ssl-config/src/test/resources/readme.md

```diff
@@ -0,0 +1,65 @@
+# Test Keys and Certificates
+
+This directory contains certificates and keys in various formats that are used to test the package.
```

Comment:
Nope; hand-crafted with love (wouldn't be too hard to script, but didn't seem worth it at the time)